app/[contentType]/[slug]/page.tsx

```diff
@@ -92,10 +92,10 @@ export default async function ContentPage({ params }: { params: Params }) {
           <time className="block text-accent-4 mb-8">
             {content.date.toString()}
           </time>
-          {content.previewImage && (
+          {Boolean(content.previewImage) && (
```

Comment:
we want to make sure that this is a Boolean when we are using a `string` in a logical `AND` expression